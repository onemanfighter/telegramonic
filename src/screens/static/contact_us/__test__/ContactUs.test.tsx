import { render } from '@testing-library/react';
import ContactUs from '../ContactUs';

describe('ContactUs screen', () => {
  it('should render correctly', () => {
    const { container } = render(<ContactUs />);

    expect(container).toMatchSnapshot();
  });
});
