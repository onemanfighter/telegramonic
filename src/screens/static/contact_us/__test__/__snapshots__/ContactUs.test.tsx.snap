// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContactUs screen should render correctly 1`] = `
<div>
  <div>
    <h1>
      Contact Us
    </h1>
  </div>
</div>
`;
